sources/packages/backend/libs/sims-db/src/entities/notification.model.ts

```diff
@@ -213,4 +213,8 @@ export enum NotificationMessageType {
    * Supporting User Information Notification.
    */
   SupportingUserInformationNotification = 29,
+  /**
+   * Student PD PPD Application Notification.
+   */
+  StudentPdPpdApplicationNotification = 30,
```

Comment:
Acronyms like PD and PPD are used in all upper cases along the solution.
Please rename it to `StudentPDPPDApplicationNotification`.
Same for all other places along the PR.